content/automate/ManageIQ/Transformation/Infrastructure/VM/rhevm.class/__methods__/checkvmininventory.rb

```diff
@@ -16,27 +16,13 @@ def main
                 destination_vm = ManageIQ::Automate::Transformation::Infrastructure::VM::RedHat::Utils.new(destination_ems).vm_find_by_name(source_vm.name)
                 raise "VM #{source_vm.name} not found in destination provider #{destination_ems.name}" if destination_vm.nil?
 
-                finished = false
-
                 destination_vm_vmdb = @handle.vmdb(:vm).where(["ems_ref = ?", destination_vm.href.gsub(/^\/ovirt-engine/, '')]).first
```

Comment:
Realize this is outside of the changes in this PR but future refactorings should be:

```ruby
destination_vm_vmdb = @handle.vmdb(:vm).find_by(:ems_ref => destination_vm.href.gsub(/^\/ovirt-engine/, ''))
```

Reply:
Changed. I suppose it has better performance.